tests/installation/installation_overview.pm

```diff
@@ -41,6 +41,20 @@ sub ensure_ssh_unblocked {
                 send_key_until_needlematch 'ssh-open', 'tab';
             }
         }
+        #performance ci need diable firewall
```

Comment:
there is a typo in here.

Can you say why the firewall would need to be disabled? Shouldn't the proper rules be added?

I recommend to move this whole block to a specific test module

Reply:
we have some network performance cases,the test scope is base on disabled firewall,and I have add a if switch get_var('DISABLE_FIREWALL') on it .